### PR TITLE
[Design] Update Typographic Scale

### PR DESCRIPTION
**Pull request by joebell93**
## Description
- Implement and test new typographic scale proposed by @mikegregory
- Includes minor fixes to various UI elements affected by our changes

Fully signed off.

## Related Issue
https://github.com/sky-uk/toolkit/issues/167

## Related PR
https://github.com/sky-uk/toolkit-core/pull/172

## Motivation and Context
We currently have too many text/heading sizes which leads to a lack of hierarchy on mobile

## How Has This Been Tested?
See private channel.

## Screenshots (if appropriate)
See private channel.

## Types of Changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Browser Support
- [x] Chrome
- [x] Edge
- [x] Firefox
- [x] IE9
- [x] IE10
- [x] IE11
- [x] Opera
- [x] Safari
- [x] Mobile Devices
- [x] Screen-readers (e.g. JAWS, Apple VoiceOver)

## Checklist
- [x] All code includes full inline documentation (as per the [template](https://github.com/sky-uk/toolkit-core/blob/master/_template.scss)).
- [x] All code conforms to the Toolkit [coding style](https://github.com/sky-uk/toolkit/wiki/Coding-Style).
- [x] All code conforms to [WCAG 2.0 level AA Accessibility Guidelines](https://www.w3.org/TR/WCAG20/).
- [x] New functions and mixins have appropriate tests.
- [x] All new and existing tests passed.
- [x] I have added instructions on how to test my changes.
- [x] Package version updated.
- [x] CHANGELOG.md updated.

### CODE REVIEW

File: components/_accordion.scss [hunk 1]
```diff
@@ -6,7 +6,7 @@ $accordion-animation-speed: $global-animation-speed !default;
 $accordion-background: transparent !default;
 $accordion-border: 1px solid color(grey-20) !default;
 $accordion-border-radius: $global-border-radius !default;
-$accordion-icon-size: 20px !default;
+$accordion-icon-size: $global-spacing-unit !default;
```
Comment (csswizardry) [inline components/_accordion.scss]: Is the icon size always going to be the same at whatever the global spacing unit is, or is it just coincidental?

Comment (joebell93) [inline components/_accordion.scss]: Just a coincidence in this case

Comment (csswizardry) [inline components/_accordion.scss]: Then we didn’t ought to use the var… if the value of `$global-spacing-unit` ever changes then our icon will break :(

Comment (joebell93) [inline components/_accordion.scss]: Good shout!

**Pull request merged**